console/package.json

```diff
@@ -7,8 +7,10 @@
     "build": "ng build",
     "prodbuild": "ng build --configuration production --base-href=/ui/console/",
     "lint": "ng lint && stylelint './src/**/*.scss' --syntax scss",
-    "e2e": "./cypress.sh run ../e2e/dev.env",
-    "e2e:open": "./cypress.sh open ../e2e/dev.env"
+    "e2e:build": "cd .. && GOOS=linux GOARCH=amd64 goreleaser build --snapshot --single-target --rm-dist --output .artifacts/zitadel/zitadel && ./e2e/docker-compose.sh build",
```

Comment:
Not many Frontend dev will have this installed. 

I think we should solve this in an other way

Reply:
trying to run it with DinD

Reply:
outscoped local builds